src/main/KeyboardControls.tsx

```diff
@@ -95,20 +88,18 @@ const Entry: React.FC<{params: KeyMapDisplayOptions}> = ({params}) => {
 
   return (
     <div css={entryStyle}>
-      <div css={labelStyle}><Trans>{params.name || t("keyboardControls.missingLabel")}</Trans></div>
-      <div css={sequencesStyle}>
-        {params.sequences.map((sequence, index, arr) => (
-          <div css={sequenceStyle} key={index}>
-            {sequence.sequence.toString().split('+').map((singleKey, index, {length}) => (
-              <>
-                <div css={singleKeyStyle} key={index}>{singleKey}</div>
-                {length - 1 !== index ? <div css={orStyle}>+</div> : ''}
-              </>
-            ))}
-            <div css={orStyle}><Trans>{arr.length - 1 !== index && t("keyboardControls.sequenceSeparator")}</Trans></div>
-          </div>
-        ))}
-      </div>
+      <div css={labelStyle}><Trans>{name || t("keyboardControls.missingLabel")}</Trans></div>
+      {sequences.map((sequence, index, arr) => (
+        <div css={sequenceStyle} key={index}>
+          {sequence.map((singleKey, index) => (
+            <>
+              <div css={singleKeyStyle} key={index}>{singleKey}</div>
+              {sequence.length - 1 !== index ? <div css={orStyle}>+</div> : ''}
```

Comment:
```suggestion
              {sequence.length - 1 !== index && <div css={orStyle}>+</div>}
```

That works too. And if you move that line above the single key div, you can just check `index > 0`.